core/templates/components/atoms/media-block-body/media-block-body.twig

```diff
@@ -1,4 +1,4 @@
-<div{{ attributes }} class="decanter-media-block__body {{ classes }}">
+<div{{ attributes }} class="decanter-media-block__body {{ modifier_class }}">
```

Comment:
The convention is to provide a `modifier_class` variable so others can inject their modifiers.